From 76a47021f77e567e5da1bc5bfab94de3472bfbce Mon Sep 17 00:00:00 2001
From: slitaz-cn <teasiu@163.com>
Date: Mon, 29 Mar 2021 21:00:25 +0800
Subject: [PATCH 08/10] set default username ecoo and password passwd
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

	修改：     package/base-files/files/etc/passwd
	修改：     package/base-files/files/etc/shadow
	修改：     package/base-files/files/usr/libexec/login.sh
	修改：     package/system/rpcd/files/rpcd.config

Signed-off-by: slitaz-cn <teasiu@163.com>
---
 package/base-files/files/etc/passwd           | 2 +-
 package/base-files/files/etc/shadow           | 2 +-
 package/base-files/files/usr/libexec/login.sh | 2 +-
 package/system/rpcd/files/rpcd.config         | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/package/base-files/files/etc/passwd b/package/base-files/files/etc/passwd
index 1d06a80361..2c980cf06f 100644
--- a/package/base-files/files/etc/passwd
+++ b/package/base-files/files/etc/passwd
@@ -1,4 +1,4 @@
-root:x:0:0:root:/root:/bin/ash
+ecoo:x:0:0:root:/root:/bin/ash
 daemon:*:1:1:daemon:/var:/bin/false
 ftp:*:55:55:ftp:/home/ftp:/bin/false
 network:*:101:101:network:/var:/bin/false
diff --git a/package/base-files/files/etc/shadow b/package/base-files/files/etc/shadow
index 4b4154f21f..d675de17b7 100644
--- a/package/base-files/files/etc/shadow
+++ b/package/base-files/files/etc/shadow
@@ -1,4 +1,4 @@
-root::0:0:99999:7:::
+ecoo:$1$V4UetPzk$CYXluq4wUazHjmCDBCqXF.:0:0:99999:7:::
 daemon:*:0:0:99999:7:::
 ftp:*:0:0:99999:7:::
 network:*:0:0:99999:7:::
diff --git a/package/base-files/files/usr/libexec/login.sh b/package/base-files/files/usr/libexec/login.sh
index 1fff39c6a0..5121506772 100755
--- a/package/base-files/files/usr/libexec/login.sh
+++ b/package/base-files/files/usr/libexec/login.sh
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-[ "$(uci -q get system.@system[0].ttylogin)" = 1 ] || exec /bin/ash --login
+[ "$(uci -q get system.@system[0].ttylogin)" = 1 ] || exec /bin/login
 
 exec /bin/login
diff --git a/package/system/rpcd/files/rpcd.config b/package/system/rpcd/files/rpcd.config
index faaed7f538..3b3997218c 100644
--- a/package/system/rpcd/files/rpcd.config
+++ b/package/system/rpcd/files/rpcd.config
@@ -3,8 +3,8 @@ config rpcd
 	option timeout 30
 
 config login
-	option username 'root'
-	option password '$p$root'
+	option username 'ecoo'
+	option password '$p$ecoo'
 	list read '*'
 	list write '*'
 
-- 
2.25.1

